web/packages/teleport/src/Users/Users.tsx

```diff
@@ -232,3 +251,39 @@ export function Users(props: State) {
     </FeatureBox>
   );
 }
+
+const InfoGuideReferenceLinks = {
+  Users: {
+    title: 'Teleport Users',
+    href: 'https://goteleport.com/docs/core-concepts/#teleport-users',
+  },
+};
+
+const InfoGuide = () => (
+  <Box>
+    <InfoTitle>Users</InfoTitle>
+    <InfoParagraph>
+      Teleport allows for two kinds of{' '}
+      <InfoExternalTextLink href={InfoGuideReferenceLinks.Users}>
```

Comment:
This link doesn't stringify correctly. Clicking it takes you to an error page.

<img width="1017" alt="image" src="https://github.com/user-attachments/assets/2cf901fb-40f0-4138-a609-7b2891037b70" />

Reply:
thanks for catching that, i correct the typing so it complains if it's not passed a string